### DISCUSSION TITLE
How to upgrade  Vitepress-Carbon ?

### DISCUSSION THREAD

**Discussion by zhang-nianqiang** [brenoepics/vitepress-carbon#40] [Q&A] [answered]
I'm a newbie so I'd like to know if there is an upgrade guidance document to facilitate my operation.

**Comment by gabriel-logan** [comment, marked as answer]
> I'm a newbie so I'd like to know if there is an upgrade guidance document to facilitate my operation.

You can simply do yarn add "dependency name" to update a dependency, same thing with npm, npm i.

**Reply by gabriel-logan** [reply]
> > I'm a newbie so I'd like to know if there is an upgrade guidance document to facilitate my operation.
> 
> You can simply do yarn add "dependency name" to update a dependency, same thing with npm, npm i.

Or you can manually change the version in package.json and then, for greater security, delete the node modules and the .lock file and then run npm install or yarn install. NEVER delete package.json.

You also have the option to install a specific version. yarn add mypackage@^x.y.z
